{"report":{"reportPrefix":"","assertionsCount":0,"lastError":{"code":"ECONNREFUSED","sessionCreate":true,"showTrace":false,"reportShown":true},"skippedAtRuntime":[],"skippedByUser":[],"skipped":[],"time":"0.002000","timeMs":2,"completed":{},"completedSections":{"__global_afterEach_hook":{"time":"0.002000","assertions":[],"commands":[],"passed":0,"errors":0,"failed":0,"skipped":0,"tests":0,"status":"fail","startTimestamp":1705575067304,"httpOutput":[["2024-01-18T10:51:07.028Z","  Request POST /session  ","{\n     capabilities: { firstMatch: [ {} ], alwaysMatch: { browserName: \u001b[32m&#39;firefox&#39;\u001b[39m } }\n  }"],["2024-01-18T10:51:07.153Z","  Request POST /session  (retry 1) ","{\n     capabilities: { firstMatch: [ {} ], alwaysMatch: { browserName: \u001b[32m&#39;firefox&#39;\u001b[39m } }\n  }"],["2024-01-18T10:51:07.269Z","  Request POST /session  (retry 2) ","{\n     capabilities: { firstMatch: [ {} ], alwaysMatch: { browserName: \u001b[32m&#39;firefox&#39;\u001b[39m } }\n  }"]],"timeMs":2,"endTimestamp":1705575067306}},"errmessages":[" \u001b[31mAn error occurred while creating a new GeckoDriver session: Connection refused to localhost:4444. If the Webdriver/Selenium service is managed by Nightwatch, check if \"start_process\" is set to \"true\".\u001b[39m"],"testsCount":0,"skippedCount":0,"failedCount":0,"errorsCount":1,"passedCount":0,"group":"","modulePath":"D:\\nest js\\nightwatch\\test\\api_testing.js","startTimestamp":"Thu, 18 Jan 2024 10:51:06 GMT","endTimestamp":"Thu, 18 Jan 2024 10:51:07 GMT","sessionCapabilities":{"browserName":"firefox","name":"Api Testing"},"sessionId":"","projectName":"","buildName":"","testEnv":"default","isMobile":false,"status":"fail","host":"localhost","name":"Api Testing","tags":[],"tests":0,"failures":0,"errors":1,"httpOutput":[["2024-01-18T10:51:07.028Z","  Request POST /session  ","{\n     capabilities: { firstMatch: [ {} ], alwaysMatch: { browserName: <span style=\"color:#0A0\">&#39;firefox&#39;<span style=\"color:#FFF\"> } }\n  }</span></span>"],["2024-01-18T10:51:07.153Z","  Request POST /session  (retry 1) ","{\n     capabilities: { firstMatch: [ {} ], alwaysMatch: { browserName: <span style=\"color:#0A0\">&#39;firefox&#39;<span style=\"color:#FFF\"> } }\n  }</span></span>"],["2024-01-18T10:51:07.269Z","  Request POST /session  (retry 2) ","{\n     capabilities: { firstMatch: [ {} ], alwaysMatch: { browserName: <span style=\"color:#0A0\">&#39;firefox&#39;<span style=\"color:#FFF\"> } }\n  }</span></span>"]],"rawHttpOutput":[["2024-01-18T10:51:07.028Z","  Request POST /session  ","{\n     capabilities: { firstMatch: [ {} ], alwaysMatch: { browserName: 'firefox' } }\n  }"],["2024-01-18T10:51:07.153Z","  Request POST /session  (retry 1) ","{\n     capabilities: { firstMatch: [ {} ], alwaysMatch: { browserName: 'firefox' } }\n  }"],["2024-01-18T10:51:07.269Z","  Request POST /session  (retry 2) ","{\n     capabilities: { firstMatch: [ {} ], alwaysMatch: { browserName: 'firefox' } }\n  }"]],"globalErrorRegister":[" \u001b[31mAn error occurred while creating a new GeckoDriver session: Connection refused to localhost:4444. If the Webdriver/Selenium service is managed by Nightwatch, check if \"start_process\" is set to \"true\".\u001b[39m"]},"name":"api_testing","httpOutput":[],"systemerr":" An error occurred while creating a new GeckoDriver session: Connection refused to localhost:4444. If the Webdriver/Selenium service is managed by Nightwatch, check if \"start_process\" is set to \"true\".\n An error occurred while creating a new GeckoDriver session: Connection refused to localhost:4444. If the Webdriver/Selenium service is managed by Nightwatch, check if \"start_process\" is set to \"true\"."}